packages/astro/src/@types/astro.ts

```diff
@@ -432,6 +432,8 @@ export interface AstroUserConfig {
 	 * @description
 	 * The base path to deploy to. Astro will build your pages and assets using this path as the root. Currently, this has no effect during development.
 	 *
+	 * You can access this value in your app via `import.meta.env.BASE_URL`.
```

Comment:
```suggestion
	 * You can alternatively access this value in your app via `import.meta.env.BASE_URL`.
```

Reply:
Im sure this would help users understand that the `import.meta.env` variables are available to them for using outside the scope of Astro files.